docs/en/your-profile-and-preferences/customise-mentions-and-other-messages.md

```diff
@@ -0,0 +1,66 @@
+---
+id: 570
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Customise mentions and other messages
+
+You can customise three different types of notifications: `Personal mentions`, `Global Mentions` and `All other messages`. For each one of them, follow the same procedure and use the table below as a reference.
+
+## Know your notification options
+
+| Option | Description | Works with |
+|:---|:---|:---|
+| Personal metions| Include messages that specifically mention you, or have your name tagged. | [Mobile and desktop] |
+| Global mentions | Include notifications for announcements or messages that are directed to the entire community. | [Mobile and desktop] |
+| All other messages| Include any messages that don't use the "personal mentions" or "global mentions".| [Mobile and desktop]
+| Same settings for all message types| Include [add a definition here]". | [Mobile and desktop] |
+
+!!! note
+	These notifications apply to communities, channels, group chats and direct messages (DMs).
+
+## Customise mentions and all other messages
+
+You can choose to be notified about mentions (personal and global) or every new message (all other messages).
+
+=== "Mobile"
+
+    1. From the tab bar, tap :mobile-communities: **Communities** to customise a community or channel, or tap :mobile-messages: **Messages** to customise a group chat or DM.
+
+    1. Long press the community, channel, group chat or DM you want to set up notifications for.
+
+    1. Tap :mobile-notifications-settings: **Notification settings**.
+
+    1. Select **Personal mentions**, **Global mentions** or **All other messages**.
+
+    1. Choose your notification preference.
+
+| Preference | Description |
+|:---|:---|
+| Notification on| Use this option to turn the notifications on. | 
+| In notification center only | Use this option to enable notifications center only. [place a link to Your Status notifications guide]. | 
+| Notifications off| Use this option to turn notifications off. | 
+
+!!! note
+	You can't customise channel notifications on desktop.
+
+## How to disable Status on-screen notifications
+
+This is an example of how you can disable notifications when you don’t want banner notifications for new messages. This may be useful when you plan to attend a public event and you don't want your notification to show on the screen.
+
+=== "Mobile"
+
+    1. From the tab bar, tap :mobile-communities: **Communities** to customise a community or channel, or tap :mobile-messages: **Messages** to customise a group chat or DM.
+
+    1. Long press the community, channel, group chat or DM you want to set up notifications for.
+
+    1. Tap :mobile-notifications-settings: **Notification settings**.
+
+    1. Turn **Same settings for all message types** on.
```

Comment:
Should be 'turn on <...>'. See the [style guide section](https://write.status.im/style-conventions/#other-style-conventions) on phrasal verbs.